mmv1/products/compute/ResizeRequest.yaml

```diff
@@ -13,21 +13,16 @@
 
 ---
 name: 'ResizeRequest'
-api_resource_type_kind: InstanceGroupManagerResizeRequest
-kind: 'compute#instanceGroupManagerResizeRequest'
```

Comment:
why are we removing the api_resource_type_kind and kind ?

Reply:
I removed them on the suggestion of [melinath](https://github.com/melinath) since they are unused according to him. Marking this comment resolved.

Reply:
`kind` is unused, but `api_resource_type_kind` is used & should be left in.

Reply:
Added api_resource_type_kind=InstanceGroupManagerResizeRequest. Since both zonal and regional refers to same mixer resource, added InstanceGroupManagerResizeRequest in both places.